src/counters.c

```diff
@@ -851,7 +851,9 @@ static void StatsLogSummary(void)
         }
     }
     SCMutexUnlock(&stats_table_mutex);
-    SCLogInfo("Alerts: %"PRIu64, alerts);
+    if (stats_enabled) {
```

Comment:
I think the check should be at the start of the function. Something like
```c
if (!stats_enabled)
   return;
```
Reason is that I think we shouldn't touch the stats internals if stats aren't enabled.

Reply:
OK. Got it.